_stbt/config.py

```diff
@@ -20,6 +20,9 @@ def get_config(section, key, default=None, type_=str):
     found, unless `default` is specified (in which case `default` is returned).
     """
 
+    if type_ == bool:
+        type_ = lambda s: s.lower() in ("1", "yes", "true", "on")
+
```

Comment:
Hmm, this looks familiar.  Ah yes see 939300a on #449.

Reply:
Cool, that implementation is nicer, I've stolen it.